spec/fixtures/ai-proxy/unit/real-stream-frames/openai/llm-v1-chat.txt

```diff
@@ -0,0 +1 @@
+data: {"choices": [{"delta": {"content": "the answer"},"finish_reason": null,"index": 0,"logprobs": null}],"created": 1711938086,"id": "chatcmpl-991aYb1iD8OSD54gcxZxv8uazlTZy","model": "gpt-4-0613","object": "chat.completion.chunk","system_fingerprint": null}
```

Comment:
As mentioned in `spec/03-plugins/38-ai-proxy/01-unit_spec.lua` file, if we don't do test for openai, do we still need to add this file?